makefile

```
@@ -7,51 +22,43 @@ tidy:
-copyenv-linux:
-	cp ./cocktails.mcp/src/.env.local ./cocktails.mcp/dist/linux/.env.local && cp ./cocktails.mcp/src/.env ./cocktails.mcp/dist/linux/.env
+copyenv:
+	@if [ -f ./cocktails.mcp/src/.env ]; then cp ./cocktails.mcp/src/.env ./cocktails.mcp/dist/linux/.env && echo 'copied .env to dist'; fi
+	@if [ -f ./cocktails.mcp/src/.env.local ]; then cp ./cocktails.mcp/src/.env.local ./cocktails.mcp/dist/linux/.env.local && echo 'copied .env.local to dist'; fi
+
+build:
+	cd ./cocktails.mcp/src && CGO_ENABLED=0 GOOS=linux go build -trimpath -ldflags "-s -w" -o ../dist/linux/cezzis-cocktails ./cmd
 
-build-linux:
-	cd ./cocktails.mcp/src && CGO_ENABLED=0 GOOS=linux go build -o ../dist/linux/cezzis-cocktails ./cmd
+compile: clean build copyenv
```

checkmake (0.2.2)

[warning] Target "compile" should be declared PHONY.

(phonydeclared)